stores/hooks.ts

```diff
@@ -39,9 +39,9 @@ export const currentUserSuffix = createSelector('currentUserSuffix', [
 export function useGlobalState<TVal>(
     initialValue: TVal,
     name: string,
-    suffix: string = useSelector(currentUserAndTeamSuffix),
```

Comment:
FYI @calebroseland. I couldn't find anywhere that we actually passed a value in for `suffix`, so I just removed the parameter entirely

Reply:
@hmhealey That's true, but I parameterized that by design for the case that a piece of global state might be desired to be cross-team or some other custom scope—this hardcodes it to `currentUserAndTeam`. I'd be 3/5 for keeping it ready to go/use as it's really small, but I understand and agree with the value in eliminating unused code/parameters 👍.

Reply:
Hopefully we won't actually need this much longer. The new redux-persist stuff I showed off a few weeks back lets us persist arbitrary parts of the store, so `setGlobalItem` and all that related infrastructure could be deprecated

Reply:
Ooh, I forgot about that—big +1 with that in mind, then. 

![image](https://user-images.githubusercontent.com/11724372/157104138-0f0c4173-b963-4f32-8e03-684d7a6ef378.png)